### ISSUE TITLE
Consolidate evaluation grids onto QuerySeqSchema with explicit window starts

### ISSUE THREAD

**Issue by gkondas** [mmcdermott/EveryQuery-conditional#26]
## Base branch

All work starts from the current tip of `feat/prevalence-weighted-boundaries`, not `main`.

Read `docs/CONDITIONAL_QUERIES.md` and `src/every_query/generate_tasks/README.md` before implementing.

## Motivation

Two evaluation-grid generators overlap:

- `sample_evaluation_query_sequences.py` emits `QuerySeqSchema` rows with one scalar label per query, but every window implicitly starts at `prediction_time`.
- `sample_evaluation_multitask_sequences.py` supports explicit starts, but also emits an all-vocabulary `.labels.npy` tensor plus a manifest, `eval_meta`, and `eval_tasks.parquet`.

Those dense evaluation labels are unnecessary. A multitask evaluation task ultimately scores only:

```text
P(target_code occurs in the final window | patient, W0, C0, A0, ..., W(K-1))
```

The earlier scalar answers are already materialized model inputs. Evaluation should use the same scalar `QuerySeqSchema` representation as the sequence model.

## Goal

Make `EQ_generate_evaluation_query_sequences` the only evaluation-grid generator:

1. Add explicit window starts to `QuerySeqSchema` evaluation grids.
2. Make `EQ_predict_multitask` consume those grids without dense all-vocabulary evaluation targets or logits.
3. Delete the redundant multitask evaluation generator and its sidecar contract.

## Issue family

- [ ] #27 — Add explicit window starts to QuerySeq evaluation grids
- [ ] #28 — Make multitask prediction consume QuerySeq evaluation grids
- [ ] #29 — Remove the legacy multitask evaluation generator

Dependencies:

- #28 starts only after #27 is complete.
- #29 starts only after both #27 and #28 are complete and green.

The dependency order is strict:

```text
QuerySeq starts
    -> multitask QuerySeq adapter + target-only scoring
    -> delete legacy multitask evaluation generator
```

The repository must not pass through a state where the old generator is deleted while `predict_multitask.py` still imports or depends on it.

## Shared semantic contract

For query position `j`:

```text
start[j] =
    prediction_time + start_duration[j]
    OR first occurrence of start_event[j] strictly after prediction_time

end[j] =
    start[j] + duration[j]
    OR first occurrence of bound_event[j] strictly after start[j]

answer[j] =
    start[j] < some occurrence of queries[j] < end[j]
```

Use the existing constants and meanings from `sample_multitask_sequences.py`:

```python
WINDOW_SEMANTICS = "open_open"
START_REFERENCE = "prediction_time"
DURATION_END_REFERENCE = "resolved_start"
MISSING_EVENT_START = "empty_window"
MISSING_EVENT_BOUNDARY = "infinity"
```

Consequences:

- Both endpoints are open.
- Duration ends are relative to the resolved start.
- A missing event start produces an empty, all-false window.
- A missing event end after a resolved start lets the window run to the end of the record.
- An unresolved start must never become an infinite positive window.
- For prediction-time starts, behavior must remain identical to the current query-sequence labeler.

## Integration decisions

- Existing query/duration/end sampling remains parity-anchored to `sample_query_sequences.py`.
- Start sampling uses the multitask sampler’s start semantics and independent RNG axes.
- This does **not** claim the complete sampled sequence distribution equals `sample_multitask_sequences.py`; controlled multitask evaluations can use `sequences_path`.
- The ordinary sequence models do not represent non-default starts. Their dataset path must accept absent/all-default starts but reject positive-duration or event starts clearly.
- The multitask prediction path must explicitly opt into active starts.
- Multitask training remains unchanged and continues using packed all-vocabulary targets.
- The final multitask evaluation prediction is the last real query in each sequence; padding must not change which query is final.

## Orchestration

This family is intended for one integration branch and one final PR, with one code-writing phase at a time:

1. Implement the first child issue.
2. Run focused tests and have a review agent inspect semantics and regressions.
3. Fix and commit.
4. Repeat for the second child.
5. Complete deletion/cleanup only after both prerequisites pass.
6. Run the complete suite and a final integration review.

Avoid concurrent writers in a shared checkout because the child issues overlap schema, dataset, predictor, model, configuration, and tests.

## Final definition of done

- `uv run pytest` passes.
- `uv run pytest -m 'slow or not slow'` passes.
- `tests/test_cli_smoke.py` passes.
- Start-at-prediction-time equivalence is pinned by a regression test.
- All three start forms pass an end-to-end hand-computed oracle, including missing-start and missing-end cases.
- Target-only multitask prediction matches the corresponding gathered full-vocabulary logits.
- No evaluation path writes or reads `.labels.npy`, a multitask manifest, `eval_meta`, or `eval_tasks.parquet`.
- The deleted CLI is absent from `pyproject.toml`.
- No live import references the deleted generator.
- Documentation describes one evaluation generator.
- `uv run pre-commit run --files <changed files>` passes without reformatting unrelated files.

## Out of scope

- Adding starts to the query-sequence **training** sampler.
- Teaching `ConditionalQueryEncoderDecoderModel` or `ConditionalQueryARModel` to encode active starts.
- Changing multitask training labels or loss.
- Merging the complete training distributions of the sequence and multitask models.
- Adding ontology support to the multitask model.
